upgrades/steps_282.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package upgrades
+
+// stateStepsFor282 returns upgrade steps for Juju 2.8.2.
+func stateStepsFor282() []Step {
```

Comment:
Where is steps_282_test.go?  Looks like we stopped writing these tests for 2.8.x?

Reply:
Yeah, the 2.8.1 ones where missing as well so I skipped them for 2.8.2. Not sure about the benefits of having these particular test suites... however, I did update the test in this package which verifies that the migration steps for 2.8.2 are registered for execution.

Reply:
@hmlanigan Ignore my previous comment. I found the tests for 2.8.1 (the test file was misspelled :D). I pushed a commit with the missing tests + correct spelling for 2.8.1 test suite.